src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -404,72 +394,74 @@ public void testGetOutputStreamOptionsMethod() throws Exception {
-                    "com.puppycrawl.tools.checkstyle.checks.naming.messages",
-                    "name.invalidPattern", new String[] {"InputMain", "^[a-z0-9]*$"},
-                    null, getClass(), null);
-            final LocalizedMessage invalidPatternMessageMainInner = new LocalizedMessage(1,
-                    "com.puppycrawl.tools.checkstyle.checks.naming.messages",
-                    "name.invalidPattern", new String[] {"InputMainInner", "^[a-z0-9]*$"},
-                    null, getClass(), null);
-            final String expectedPath = getFilePath("InputMain.java");
-            assertEquals("Unexpected output log", auditStartMessage.getMessage() + EOL
-                    + "[ERROR] " + expectedPath + ":3:14: "
-                    + invalidPatternMessageMain.getMessage() + " [TypeName]" + EOL
-                    + "[ERROR] " + expectedPath + ":5:7: "
-                    + invalidPatternMessageMainInner.getMessage() + " [TypeName]" + EOL
-                    + auditFinishMessage.getMessage() + EOL, systemOut.getLog());
-            assertEquals("Unexpected system error log",
-                    errorCounterTwoMessage.getMessage() + EOL, systemErr.getLog());
-        });
-        Main.main("-c",
-                getPath("InputMainConfig-classname2-error.xml"),
+    public void testExistingTargetFilePlainOutput(@SysErr Capturable systemErr,
+            @SysOut Capturable systemOut) throws IOException {
+        systemErr.capture();
+        systemOut.capture();
+        Main.main("-c", getPath("InputMainConfig-classname.xml"), "-f", "plain",
                 getPath("InputMain.java"));
+        assertEquals(auditStartMessage.getMessage() + EOL + auditFinishMessage.getMessage() + EOL,
+                systemOut.getCapturedData(), "Unexpected output log");
+        assertEquals("", systemErr.getCapturedData(), "Unexpected system error log");
+    }
+
+    @Test
+    public void testExistingTargetFileWithViolations(@SysErr Capturable systemErr,
+            @SysOut Capturable systemOut) throws IOException {
+        systemErr.capture();
+        systemOut.capture();
+        Main.main("-c", getPath("InputMainConfig-classname2.xml"), getPath("InputMain.java"));
+        final LocalizedMessage invalidPatternMessageMain = new LocalizedMessage(1,
+                "com.puppycrawl.tools.checkstyle.checks.naming.messages",
+                "name.invalidPattern", new String[] {"InputMain", "^[a-z0-9]*$"},
+                null, getClass(), null);
+        final LocalizedMessage invalidPatternMessageMainInner = new LocalizedMessage(1,
+                "com.puppycrawl.tools.checkstyle.checks.naming.messages",
+                "name.invalidPattern", new String[] {"InputMainInner", "^[a-z0-9]*$"},
+                null, getClass(), null);
+        final String expectedPath = getFilePath("InputMain.java");
+        assertEquals(auditStartMessage.getMessage() + EOL
```

Comment:
EOL formatting can be changed to use AbstractPathTestSupport#addEndOfLine method to make it easier to read

Reply:
done. This also had to be done separately, but they are few.